stdlib/moreLabels.mli

```diff
@@ -194,5 +200,7 @@ module Set : sig
       val find_last_opt: f:(elt -> bool) -> t -> elt option
       val of_list: elt list -> t
     end
-  module Make : functor (Ord : OrderedType) -> S with type elt = Ord.t
+  module Make : functor (Ord : OrderedType) -> S
+    with type elt = Ord.t
+     and type t = Set.Make(Ord).t
```

Comment:
These signatures have been altered in order to get the stdlabels test to pass - should we also be doing this to the unlabelled variants, though?

You'll have a much better intuition for this than me - are there repercussions to this change? (in other words, should it be in a commit, or even possibly a pull request, of its own?)